### PR TITLE
fix bugs and add docs for `Process`

### PR DESCRIPTION
**Pull request by yalinli2**
@joyxyz1994 Take a look and merge if you don't see problems? Feel free to update - you are much more familiar with this class than me 😊 Thanks!

### CODE REVIEW

File: qsdsan/utils/parsing.py [hunk 1]
```diff
@@ -156,7 +156,8 @@ def get_stoichiometric_coeff(reaction, ref_component, components, conserved_for,
         coeff_dct = symbolize(coeff_dct, components, conserved_for, parameters)
         coeff = dct2list(coeff_dct, components)
     if ref_component:
-        normalize_factor = abs(coeff[components._index[ref_component]])
+        try: normalize_factor = abs(coeff[components.index(str(ref_component))])
```
Comment (joyxyz1994) [inline qsdsan/utils/parsing.py:159]: Do we still need the breakpoint here?

**Comment by joyxyz1994** [comment]
@yalinli2 this looks good! I'll go ahead and merge. Thanks!!

**Comment by yalinli2** [comment]
Ah definitely no!

File: qsdsan/utils/parsing.py (hunk 1 above)
Comment (yalinli2) [reply qsdsan/utils/parsing.py:159]: Took it out in 48bab7283dd35b9cc56cc97f3715c9241f1aef97, thanks!

**Pull request merged**